src/libs/E2E/tests/openSearchRouterTest.e2e.ts

```diff
@@ -59,7 +63,7 @@ const test = () => {
             if (entry.name === CONST.TIMING.SEARCH_ROUTER_RENDER) {
                 E2EClient.submitTestResults({
                     branch: Config.E2E_BRANCH,
-                    name: 'Open Search Router TTI',
+                    name: `${name} Open Search Router TTI`,
```

Comment:
NAB. There are still other tests not using the `name` variable

Reply:
Which one? 👀

Reply:
There are 2 calls above this that use `Open Search Router TTI`

Reply:
https://github.com/Expensify/App/blob/eeb623d4265e053898a5ee1d56c758a0a261bf82/src/libs/E2E/tests/openSearchRouterTest.e2e.ts#L43-L56